erreser.go

```
@@ -0,0 +1,624 @@
+func deleteFile(file string) {
+	log.Printf("Deleting file: %s", file)
+	err := os.Remove(file)
+	if err != nil {
+		log.Fatal(err)
+	}
+	log.Printf("Deleted file: %s", file)
+}
+
+func publishToTheChannel(feed models.Feed, item models.Item, episodeFile string) {
+	log.Printf("Publishing to telegram %s", item.Title)
+	log.Printf("Published %s", item.TgPublished)
```

golangci-lint (1.62.2)

printf: log.Printf format %s has arg item.TgPublished of wrong type int

(govet)